src/serialization/sb2_specmap.js

```diff
@@ -1110,9 +1110,8 @@ const specMap = {
         opcode: 'sensing_current',
         argMap: [
             {
-                type: 'input',
-                inputOp: 'sensing_currentmenu',
```

Comment:
So I could just be confused about this, but it seems weird that we're not making use of the `sensing_currentmenu` anymore. Is this just a piece of code where we originally thought `sensing_currentmenu` was going to be a dynamic menu but it isn't?

Reply:
I think so yeah. I didn't find any sensing_currentmenu in scratch-blocks.